taxonomies/categories.txt

```diff
@@ -25874,13 +25986,18 @@ fr:Rocamadour
 country:en:France
 wikidata:en:Q768189
 oqali_family:en: fr:Fromages - Crottin de chevre
+ciqual_food_code:en:12847
+ciqual_food_name:en:Rocamadour cheese, from goat's milk
 
 <en:Goat cheeses
 <en:Soft cheese with a natural rind
 <en:French cheeses
+en:Sainte Maure cheese from goat's milk
```

Comment:
```suggestion
en:Sainte-Maure de Touraine, Sainte Maure cheese
```
It's a PDO, so better keepthe complete name in English

Reply:
resolu